biz.aQute.launcher/src/org/osgi/framework/connect/ConnectContent.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) OSGi Alliance (2019). All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.osgi.framework.connect;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+import java.util.Optional;
+public interface ConnectContent {
```

Comment:
I think it is wrong for us to have a copy of the package's source code in our repo. We can easily miss important changes as the OSGi spec changes in development. We should instead reference the OSGi snapshot jar to access the package. https://oss.sonatype.org/content/repositories/snapshots/org/osgi/org.osgi.framework/1.10.0-SNAPSHOT/

Reply:
I wanted to placate you :-) I talked to Thomas and Karl and they seemed rather anxious in this area. But if we can import this snapshot, that would be great.

Reply:
Hmm, but then we take all Framework packages from the snapshot?

Reply:
If we have the snapshot jar following the released jar on the `-buildpath`, and we use `-split-package:=first` for the package instruction, we will get the released packages except for connect which is not in the released jar, no?

Reply:
It is only used to build, the API is not ending up in any binary so split package is not needed. I made it the second entry so this is perfect! Much better than embedded these sources.